mitype/app.py

```diff
@@ -278,6 +286,12 @@ def update_state(self, win):
 
             win.addstr(" to see a replay! ")
 
+            win.addstr(self.line_count + 4, 0, " Press ")
+
+            win.addstr(" TAB ", curses.color_pair(5))
+
+            win.addstr(" to try a new test! ")
```

Comment:
This message could be better writtten as "to retry".

Reply:
Updated the message.